src/test/java/org/folio/rest/api/RequestExpirationApiTest.java

```diff
@@ -42,24 +42,13 @@ public void checkIdsAfterEach() {
   }
 
   @Test
-  public void canExpireASingleOpenUnfilledRequest()
-    throws InterruptedException,
-    MalformedURLException,
-    TimeoutException,
-    ExecutionException {
+  public void canExpireASingleOpenUnfilledRequest() throws InterruptedException, MalformedURLException, TimeoutException,
+      ExecutionException {
     UUID id = UUID.randomUUID();
     UUID itemId = UUID.randomUUID();
 
-    createEntity(
-      new RequestRequestBuilder()
-      .hold()
-      .withId(id)
-      .withRequestExpiration(new DateTime(2017, 7, 30, 10, 22, 54, DateTimeZone.UTC))
-      .withItemId(itemId)
-      .withPosition(1)
-      .withStatus(OPEN_NOT_YET_FILLED)
-      .create(),
-      requestStorageUrl());
+    createEntity(new RequestRequestBuilder().hold().withId(id).withRequestExpiration(new DateTime(2017, 7, 30, 10, 22, 54,
```

Comment:
Can you help me understand why these lines have been reformatted?

I personally find it much easier to interpret the relevant details of the records with the multi line layout.

Reply:
reformatted after applying import ordering with Eclipse Code Formatter
reverting this part

Reply:
reverted